tardis/io/tests/test_model_reader.py

```diff
@@ -5,31 +5,35 @@
 import pytest
 
 import tardis
-from tardis.io.model_reader import (read_artis_density, 
-read_simple_ascii_abundances)
+from tardis.io.model_reader import read_artis_density, \
```

Comment:
Sorry we do not accept PEP8 fixes at this time.
On Fri, 4 Mar 2016 at 09:05, Anshuman Suri <notifications@github.com> wrote:

> In tardis/io/tests/test_model_reader.py
> <https://github.com/tardis-sn/tardis/pull/504#discussion_r55000528>:
>
> > @@ -5,31 +5,35 @@
> >  import pytest
> >
> >  import tardis
> > -from tardis.io.model_reader import (read_artis_density,
> > -read_simple_ascii_abundances)
> > +from tardis.io.model_reader import read_artis_density, \
>
> Fair enough. I'll make the appropriate changes
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tardis-sn/tardis/pull/504/files#r55000528>.
>
-- 
Dr. Wolfgang E Kerzendorf
ESO Fellow
European Southern Observatory
Karl-Schwarzschild Str. 2, 85748
Garching bei Muenchen, Germany